### ISSUE TITLE
Images manquantes

### ISSUE THREAD

**Issue by doudz** [Jeedom-Zigate/jeedom-plugin-zigate#118]
Liste des images à ajouter :
- [x] lumi.remote.b286acn01
- [x] TH-T_V15
- [x] lumi.ctrl_ln1.aq1
- [ ] ~~122-130-1/2-2~~
- [x] TRADFRI_bulb_GU10_WS_400lm
- [x] ~~lumi.sensor_swit~~
- [ ] Dimmer_switch_w/o_neutral
- [ ] lumi.remote.b186acn01
- [ ] PS-SPRZMS-01

**Comment by llaumgui** [comment]
@doudz tu es sûr du lumi.sensor_swit ? C'est pas le lumi.sensor_switch ? Qui lui est bien présent...
https://jeedom-zigate.github.io/jeedom-plugin-zigate/fr_FR/xiaomi

**Comment by llaumgui** [comment]
Par contre TH-T_V15 & 122-130-1/2-2 c'est quoi la marque ?

**Comment by doudz** [comment]
Pour lumi.sensor_swi c'est peut-être une erreur, on va l'ignorer pour le moment.
Pour TH-T_V15 la marque c'est Heiman
Par contre pour 122-130-1/2-2 je ne sais pas

**Comment by doudz** [comment]
On va l'ignorer pour le moment, j'ai pas assez d'info

**Comment by doudz** [comment]
Je rouvre car j'ajoute des petits nouveaux

**Comment by doudz** [comment]
J'en ajoute encore...

**Comment by antfleu** [comment]
Peut être volet profalux :)

**Comment by doudz** [comment]
@antfleu malheureusement pour les Profalux ça ne sera pas possible car il ne possède pas model identifier (le type) donc on ne peut pas les reconnaitre avec certitude

**Comment by doudz** [comment]
J'en ajoute encore, cette liste est sans fin...
Je me demande si on devrait pas procéder autrement, poser les images dans le dossier et laisser le plugin les récupérer de temps en temps directement sur github sans forcement besoin de mettre à jour

**Comment by llaumgui** [comment]
Après on peut mettre un tag need help :-)

**Comment by Amigagame** [comment]
et pourquoi ne pas permettre à chaque utilisateur d'uploader (attacher) l'image qu'il veut sur chaque équipement ?

**Comment by doudz** [comment]
Il est possible de copier/coller les images ici sans problème, on pourra ainsi les récupérer facilement

**Comment by paulnux** [comment]
Pour les images comme ça cela vous va ?
TRADFRI_control_outlet:
https://cdn.ikea.com.tr/urunler/500_500/PE640306.jpg

**Comment by Amigagame** [comment]
Image pour l'ampoule lumi.light.aqcn02

[xiaomi_aqara_smart_led_light_bulb_e27_9w_wifi_app_remote_control__zp3030390101140_1_](https://user-images.githubusercontent.com/14044648/52947253-7a265200-3376-11e9-8dea-97f6f8da2b0e.jpg)

**Comment by paulnux** [comment]
TRADFRI_motion_sensor
https://cdn.ikea.com.tr/urunler/500_500/PE695174.jpg
ou
https://cdn.ikea.com.tr/urunler/500_500/PE582930.jpg

**Comment by paulnux** [comment]
TRADFRI_transformer_10W
https://www.ikea.com/PIAimages/0625251_PE692159_S3.JPG
ou
https://www.ikea.com/PIAimages/0587550_PE672682_S3.JPG

**Comment by paulnux** [comment]
Pour moi TRADFRI_transformer_10W et TRADFRI_Driver_10W c'est pareil
https://www.ikea.com/us/en/catalog/products/10356189/
https://www.ikea.com/fr/fr/catalog/products/50356187/

**Comment by doudz** [comment]
Merci

**Comment by paulnux** [comment]
TRADFRI_bulb_E14_W_op/ch_400lm
https://www.ikea.com/fr/fr/images/products/tradfri-ampoule-a-led-e-lumen-blanc__0609532_PE684585_S4.JPG

**Comment by paulnux** [comment]
TRADFRI_bulb_E27_WS_opal_1000lm
https://www.ikea.com/fr/fr/images/products/tradfri-ampoule-led-e-lumen-blanc__0475690_PE615731_S4.JPG

**Comment by paulnux** [comment]
LIGHTIFY_Outdoor_Flex_RGBW
https://cdn.lumstatic.com/catalog/product/thumbnail/25121d744882335e740bd2d9adcb8bf4/image/2000x2000/000/80/3/7/371a5cca4729eac4e1ec34b82c8f84543f64724977f4ad9008651b4fcf511c4a.jpg

**Comment by Deepcore93** [comment]
LLC020 : 
![llc020](https://user-images.githubusercontent.com/42214914/53225235-a5ab8400-3677-11e9-8855-55c5e37db6d4.jpg)
LST001 : 
![lst001](https://user-images.githubusercontent.com/42214914/53225257-b6f49080-3677-11e9-8bd3-7e859a00acf5.jpg)
LTW007 : 
![ltw007](https://user-images.githubusercontent.com/42214914/53225266-c07df880-3677-11e9-91d1-e71401ff99a6.jpg)

**Comment by paulnux** [comment]
RB_278_T
https://cdn.tink.de/cdn/839903/media/catalog/product/cache/1/small_image/640x/17f82f742ffe127f42dca9de82fb58b1/i/n/innr_led_e27_lampe_-_dimmbar_wei_zigbee_3.0_rb_278_t-2_doppelpack_1.jpg

**Comment by Deepcore93** [comment]
GL-C-006 : 
![gl-c-006](https://user-images.githubusercontent.com/42214914/53389724-ad796a00-3990-11e9-8392-9a17d22d1743.png)

**Comment by Deepcore93** [comment]
GL-D-003Z :
![gl-d-003z](https://user-images.githubusercontent.com/42214914/53389853-4ad49e00-3991-11e9-8a6e-121162e42b88.jpg)

**Comment by paulnux** [comment]
FB56-ZCW08KU1.1
https://i2.wp.com/zigate.fr/wp-content/uploads/2017/11/jiawen_ampoule_rgb.jpg?resize=300%2C300&ssl=1

**Comment by paulnux** [comment]
@doudz 
Tu n'a pas coché RB_278_T
Un soucis avec l'image ?

**Comment by nono2032** [comment]
pas dans la liste mais si tu peux l'ajouter 👍 
ZG9101SAC-HP
![image](https://user-images.githubusercontent.com/47389767/53435534-4fcd3800-39f9-11e9-83f5-b18d1f024163.png)

**Comment by doudz** [comment]
@paulnux oui, j'aurai préféré une seule ampoule sur l'image. Il faut que je la découpe

**Comment by paulnux** [comment]
@doudz  Pas la peine que tu découpe voici un lien avec une seule ampoule:
https://www.hellosmart.nl/media/catalog/product/cache/d73a5018306142840707bd616a4ef293/r/b/rb_178_t_bulb_lights_on.jpg

**Comment by battousai90** [comment]
Bonjour,

Je possède un panneau LED Ikea Floalt, il marche parfaitement.
https://www.ikea.com/be/fr/produits/luminaires/appliques-murales/floalt-panneau-led-connect%C3%A9-intensit%C3%A9-lumineuse-r%C3%A9glable-spectre-blanc-art-40332210/

**Comment by nono2032** [comment]
Inter 1 sortie Zemismart
FNB56-ZSW01LX2.0

![FNB56-ZSW01LX2 0](https://user-images.githubusercontent.com/47389767/54077517-e0d5c600-42b9-11e9-9389-619f403fec55.JPG)

**Comment by paulnux** [comment]
lumi.remote.b186acn01
https://img.myefox.fr/img2/images/20190103/_ceaa706ed143f74e24b7eb9a651d2de4.images.650x650.jpg

**Comment by paulnux** [comment]
@doudz   l'image du lumi.remote.b186acn01 convient ou pas ?

**Comment by doudz** [comment]
> @doudz l'image du lumi.remote.b186acn01 convient ou pas ?

oui :+1:

**Comment by doudz** [comment]
> Bonjour,
> 
> Je possède un panneau LED Ikea Floalt, il marche parfaitement.
> https://www.ikea.com/be/fr/produits/luminaires/appliques-murales/floalt-panneau-led-connect%C3%A9-intensit%C3%A9-lumineuse-r%C3%A9glable-spectre-blanc-art-40332210/

il me faudrait le "type" exact, merci

**Comment by jlagrut01** [comment]
sirene Heiman
https://images.app.goo.gl/A3j2aUkv8NVm462fA

**Comment by deepee2m** [comment]
LEDVANCE Smart+ Prise Connectée ZigBee
Type : Plug Z3
https://images-na.ssl-images-amazon.com/images/I/51uBHwM%2Bq9L._AC_SL1000_.jpg
